src/applications/claims-status/components/claim-document-request-pages/Automated5103Notice.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+import { buildDateFormatter } from '../../utils/helpers';
+
+function Automated5103Notice({ item }) {
+  const formattedDueDate = buildDateFormatter()(item.suspenseDate);
+  if (item.displayName !== 'Automated 5103 Notice Response') {
+    return null;
+  }
+  return (
+    <div id="automated-5103-notice-page">
+      <h1 className="vads-u-margin-top--0 vads-u-margin-bottom--2">
+        Review the list of evidence we need
+      </h1>
+      <p>
+        We sent you a “5103 notice” letter that lists the types of evidence we
+        may need to decide your claim.
+      </p>
+      <Link
```

Comment:
We don't need to worry about this in this PR, but at some point we should make this a component, I feel like we use this code in at least 3 different places

Reply:
Are you thinking that we should make the link a component?

Reply:
created this ticket in regards to @jerekshoe's comment https://github.com/department-of-veterans-affairs/va.gov-team/issues/85537